Update  Jakarta Persistence Section annotation (#461)
jakartaee/data: spec/src/main/asciidoc/portability.asciidoc

==== Jakarta Persistence Annotations

> Jakarta Data, when used in conjunction with a Jakarta Data provider that is backed by Jakarta Persistence, requires support for the jakarta.persistence.Entity annotation on entity classes and requires support for all annotations in the Jakarta Persistence specification.

Reviewer: Missing backticks around `jakarta.persistence.Entity`.

> Jakarta Data, when used in conjunction with a Jakarta Data provider that is backed by Jakarta Persistence, requires support for the `jakarta.persistence.Entity` annotation on entity classes and requires support for all annotations in the Jakarta Persistence specification.

Reviewer: ```suggestion
A Jakarta Data provider backed by Jakarta Persistence must allow the use of `jakarta.persistence.Entity` as an entity-defining annotation.
```
